src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -390,6 +461,7 @@ public String toString() {
     public String toStringDetails() {
         String retStr = "Problem[";
         
+        //basic configuration settings
```

Comment:
First comment:  I don't think Apache Commons is currently part of our vendor library; making this change would require adding that library, which I think it rather large and seems like unwarranted bloat for such a small gain.
Second comment: I'll assume that since you said "Someday", you don't consider this an item that's necessary to fix now to get the PR approved.
On that basis, I'm marking the comment "Resolved".